.github/workflows/cicd.yml

```
@@ -183,7 +183,7 @@ jobs:
     strategy:
       matrix:
         python: ["3.6", "3.7", "3.8", "3.9", "pypy3"]
-        twisted: ["19.2", "current"]
+        twisted: ["19.2", "current", "trunk"]
```

Maybe disable early failing that is causing cancellation and skipping of remaining jobs from the matrix.

```suggestion
        twisted: ["19.2", "current", "trunk"]
        fail-fast: false
```